event-land/scss/layout/_top-bar.scss

```diff
@@ -0,0 +1,58 @@
+// Variabels (only used in this file)
+$_itemsPadding: $space3;
```

Comment:
why you just didn't use `$space3` instead of add that into another variable?

Reply:
At first I thought I would use this $_itemsPadding variable in some more places and I wanted them to be the same and sync together, In that case if I had used for example $space3 and then wanted to change it to $space4 I would need to change it in all the places too that's why I used $_itemsPadding to make them all sync But in this case now I think about it I have only used it in 2 places and that does not make any sense to have this variable **so I will remove it**